.buildkite/ui-test-demo-app.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -eu
+
+echo "--- :ruby: Setup Ruby tooling"
+install_gems
+
+echo "--- :cocoapods: Install Pods"
+install_cocoapods
+
+echo "--- :calling: UI Test Demo App"
+set +e
+bundle exec fastlane ui_test_demo_app
+TESTS_EXIT_STATUS=$?
+set -e
+
+if [[ "$TESTS_EXIT_STATUS" -ne 0 ]]; then
+  # Keep the (otherwise collapsed) current "Testing" section open in Buildkite logs on error. See https://buildkite.com/do s/pipelines/managing-log-output#collapsing-output
+  echo "^^^ +++"
+  echo "UI Tests failed!"
+
+  echo "--- :camera_with_flash: Extracting Screenshots"
+  brew install chargepoint/xcparse/xcparse
+  xcparse screenshots fastlane/test_output/UITests.xcresult .build/screenshots/
```

Comment:
During development, the Google SignIn rendered in Safari was in Italian, because I'm currently in Italy, so I used _and commited_ Italian copy.

Then forgot all about it.

When the tests failed, I couldn't understand why, so I started working on uploading the failure screenshots.

By the time I was done, I had realized my mistake. Still, I'm sure this will be useful in the future.

Reply:
There are quite a few requirements and parameters that go into this approach, such as the prefix used in the code, the `.xcresult` name, and the fact that there are screenshots in the first place. Still, I can see this being extracted in our Buildkite plugin and adopted across the board.